src/utils.ts

```diff
@@ -87,3 +88,34 @@ export function makePromiseCancelable<T>(
   };
   return cancelablePromise as CancelablePromise<T>;
 }
+
+export function convertDateToTimestamp(date: Date) {
+  return date.getTime();
```

Comment:
the name of the function is longer than it's actual code xD
can't we just do `date.getTime()` instead of `convertDateToTimestamp(date)` :) ?

Reply:
Hehe. True.